package.json

```diff
@@ -25,6 +25,7 @@
     "@aws-sdk/client-ses": "^3.76.0",
     "@types/express": "^4.17.13",
     "@types/node": "^18.7.16",
+    "all": "^0.0.0",
```

Comment:
can this be deleted then, since we dont need it?

Reply:
yes. I don't remember installing this one.

Reply:
removed in latest commit to branch! Was accidental